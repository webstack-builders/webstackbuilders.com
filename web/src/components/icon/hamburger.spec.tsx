import React from 'react'
import renderer from 'react-test-renderer'

import HamburgerIcon from './hamburger'

describe('HamburgerIcon', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<HamburgerIcon />)
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
