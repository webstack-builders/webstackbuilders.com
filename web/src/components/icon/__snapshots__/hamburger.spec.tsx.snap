// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HamburgerIcon renders correctly 1`] = `
<svg
  fill="none"
  height="1em"
  preserveAspectRatio="xMidYMid"
  viewBox="0 0 25 25"
  width="1em"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M5 7.5H20"
    stroke="currentColor"
    style={
      Object {
        "vectorEffect": "non-scaling-stroke",
      }
    }
  />
  <path
    d="M5 12.5H20"
    stroke="currentColor"
    style={
      Object {
        "vectorEffect": "non-scaling-stroke",
      }
    }
  />
  <path
    d="M5 17.5H20"
    stroke="currentColor"
    style={
      Object {
        "vectorEffect": "non-scaling-stroke",
      }
    }
  />
</svg>
`;
